roles/kafka_connect/tasks/deploy_connectors.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Register Kafka Connect Subgroups
-  set_fact: subgroups="{{ ((subgroups | default([])) + hostvars[item].group_names) | difference('kafka_broker, ksql, kafka_connect, kafka_rest, kerberos, ksql, schema_registry, zookeeper, kafka_broker_parallel, ksql_parallel, kafka_connect_parallel, kafka_rest_parallel, kerberos_parallel, ksql_parallel, schema_registry_parallel, zookeeper_parallel') }}"
+  set_fact: subgroups="{{ ((subgroups | default([])) + hostvars[item].group_names) | difference('kafka_broker, ksql, kafka_connect, kafka_rest, kerberos, ksql, schema_registry, zookeeper, kafka_broker_parallel, ksql_parallel, kafka_connect_parallel, kafka_rest_parallel, kerberos_parallel, ksql_parallel, schema_registry_parallel, zookeeper_parallel, kafka_controller, kafka_controller_parallel') }}"
```

Comment:
we have not added *serial for any component here, I think the reason must be the fact that we only run the components in paralllel for deployment. Serial only works in upgrades and this task is for connector deployment

Reply:
for replicator I think it must be because it's not a cp component per say, we have created a new role to help customers deploy replicator easily but it is essentially a part of Connect, similar to connector